financeager/fflask.py

```diff
@@ -63,5 +63,9 @@ def create_app(data_dir=None, config=None):
         EntryResource,
         "{}/<period_name>/<table_name>/<eid>".format(PERIODS_TAIL),
         resource_class_args=(server,))
+    api.add_resource(
+        VersionResource,
+        '/version'
```

Comment:
Can you introduce a constant `VERSION_TAIL` in `financeager/__init__.py` for the `/version` part; and also substitute the use in the `httprequests` module?